protobuf/lib/src/protobuf/field_info.dart

```diff
@@ -5,7 +5,8 @@
 part of protobuf;
 
 /// An object representing a protobuf message field.
-class FieldInfo<T> {
+/// For a repeated field S is List<T> for a non-repeated field S=T.
```

Comment:
I see some type parameters S and U throughout the PR. Are they meant to be separate types? The comment above mentions S, but the parameter is U here. I am guessing either is meant to represent the List<T> as mentioned

Reply:
Sorry - I did a rename, but forgot to rename the comment.